custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSearch.kt

```diff
@@ -69,26 +69,7 @@ class WorkOnSearch(
     private fun openGlobalIssueSearch(): IssueNavigator {
         val querryList: MutableList<String> = ArrayList()
         querryList.add("/issues/?jql=order%20by%20created%20DESC")
-        querryList.add("/issues/?jql=project%20%3D%20ACISDATA%20order%20by%20key%20DESC")
-        querryList.add("/issues/?jql=project%20%3D%20ACISDATA%20AND%20" +
-            "resolution%20%3D%20Fixed%20AND%20assignee%20in%20(EMPTY)%20order%20by%20updated%20DESC%20%2C%20key%20DESC")//assig
-        querryList.add("/issues/?jql=order%20by%20lastViewed%20DESC%2C%20key%20DESC")
-        querryList.add("/issues/?jql=reporter%20in%20(EMPTY)%20order%20by%20created%20DESC%2C%20key%20DESC")
-        querryList.add("/issues/?jql=project%20in%20(GCSDA%2C%20BCI1)%20AND%20status%20in%20(Accepted%2C%20Done)%20" +
-            "AND%20assignee%20in%20(membersOf(acis_dev)%2C%20EMPTY)%20order%20by%20key%20DESC")
-        querryList.add("/issues/?jql=")
-        querryList.add("/issues/?jql=project%20in%20(ACISDATA%2C%20AMSP%2C%20GLOCOMMDIS)%20" +
-            "AND%20status%20not%20in%20(%20Resolved)%20" +
-            "AND%20creator%20in%20(EMPTY)%20order%20by%20key%20DESC%2C%20created%20DESC")
-        querryList.add("/issues/?jql=issuetype%20in%20(Bug%2C%20Change)%20" +
-            "AND%20assignee%20is%20EMPTY%20AND%20\"Assignee%20Group\"%20%3D%20administrators%20" +
-            "and%20status%20not%20in%20(Done%2C%20Cancelled)%20ORDER%20BY%20issuetype%20ASC%2C%20" +
-            "priority%20DESC%2C%20created%20DESC%2C%20status%20ASC%2C%20lastViewed%20DESC")
-        querryList.add("/issues/?jql=Application%20%3D%20\"Access%20Management\"%20" +
-            "and%20assignee%20%3D%20557057%3A68425415-a8ec-439d-a05b-0b03720dcadf%20" +
-            "and%20status%20in%20(Done)%20or%20assignee%20is%20empty%20" +
-            "and%20component%20in%20(.NET)%20AND%20\"Reporting%20Country\"%20%3D%20Canada%20" +
```

Comment:
This looks like UGC. Please never commit any user generated content to the repo!